src/containers/MultidisciplinarySubject/MultidisciplinarySubjectPage.jsx

```diff
@@ -64,9 +64,9 @@ const MultidisciplinarySubjectPage = ({ match, history, location }) => {
 
   const items = topics.map(topic => ({
     title: topic.name,
-    introduction: topic.meta.metaDescription,
-    image: topic.meta.metaImage.url,
-    imageAlt: topic.meta.metaImage.alt,
+    introduction: topic.meta?.metaDescription,
+    image: topic.meta?.metaImage?.url,
+    imageAlt: topic.meta?.metaImage?.alt,
```

Comment:
både meta og metaImage er optional?

Reply:
Ikkje egentlig men det er en feil på en artikkel i prod der metabildet ikkje er satt. Egentlig burde vi vist visuelt element her i stedet for metabilde. Men det får bli neste runde.

Reply:
Kunne klart meg med ? etter metaImage.

Reply:
ok